Storage/Storage/Model/Copiable/Models+Copiable.generated.swift

```diff
@@ -25,7 +25,7 @@ extension Storage.GeneralAppSettings {
         feedbacks: CopiableProp<[FeedbackType: FeedbackSettings]> = .copy,
         isViewAddOnsSwitchEnabled: CopiableProp<Bool> = .copy,
         isInAppPurchasesSwitchEnabled: CopiableProp<Bool> = .copy,
-        isTapToPayOnIPhoneSwitchEnabled: CopiableProp<Bool> = .copy,
```

Comment:
I'm wondering, why do we replace `isTapToPayOnIPhoneSwitchEnabled` with this new property? Or is it that we take advantage of this PR to remove the previous `isTapToPayOnIPhoneSwitchEnabled`?

Reply:
Just cleaning up the unused switch storage property at the same time. The switch was already removed, and the stored setting isn't used anywhere.